src/components/UserInfo.js

```diff
@@ -15,5 +15,13 @@ export default class UserInfo {
   setUserInfo(userInf) {
     this._profileName.textContent = userInf.name;
     this._jobElement.textContent = userInf.job;
+    if (userInf.avatar) {
+      this._avatar.src = userInf.avatar;
```

Comment:
`      this._avatar.src = userInf.avatar;
`

Please, use method `setAvatar` here not to duplicate the code